gravitee-am-plugins-handlers/gravitee-am-plugins-handlers-certificate/src/main/java/io/gravitee/am/plugins/certificate/plugin/CertificatePluginHandler.java

```diff
@@ -31,11 +35,31 @@ public class CertificatePluginHandler extends AmPluginHandler<Certificate<?, ?>>
 
     private final Logger LOGGER = LoggerFactory.getLogger(CertificatePluginHandler.class);
 
+    @Autowired
+    private PluginConfigurationValidatorsRegistry validatorsRegistry;
+
     @Override
     public boolean canHandle(Plugin plugin) {
         return type().equalsIgnoreCase(plugin.type());
     }
 
+    @Override
+    protected void handle(Plugin plugin, Class<?> pluginClass) {
+        super.handle(plugin, pluginClass);
+        if (pluginManager.findById(plugin.id()) != null) {
+            registerValidator(plugin);
+        }
```

Comment:
**Thought**: I would move it to the AmPluginHandler (with the registerValidator) so that all plugin can take benefit of this validator. (obviously we will still have to adapt the other plugin beans)

Reply:
I would handle it in https://gravitee.atlassian.net/browse/AM-3373. For now, its strict "Certificate" feature.
WDYT?